src/genericitems/serialisers.cpp

```diff
@@ -0,0 +1,199 @@
+        const auto &v = std::any_cast<const Array<SampledDouble> &>(a);
+        if (!parser.writeLineF("{}\n", v.nItems()))
+            return false;
+        for (auto n = 0; n < v.nItems(); ++n)
+            if (!v.at(n).serialise(parser))
+                return false;
+        return true;
+    });
+    registerSerialiser<Array<Vec3<double>>>([](const std::any &a, LineParser &parser, const CoreData &coreData) {
+        const auto &v = std::any_cast<const Array<Vec3<double>> &>(a);
+        if (!parser.writeLineF("{}\n", v.nItems()))
+            return false;
+        for (auto n = 0; n < v.nItems(); ++n)
+            if (!parser.writeLineF("{}\n", v.at(n).x, v.at(n).y, v.at(n).z))
+                return false;
+        return true;
+    });
+    registerSerialiser<Array2D<char>>([](const std::any &a, LineParser &parser, const CoreData &coreData) {
+        const auto &v = std::any_cast<const Array2D<char> &>(a);
+        if (!parser.writeLineF("{}  {}  {}\n", v.nRows(), v.nColumns(), DissolveSys::btoa(v.halved())))
+            return false;
+        for (auto &n : v)
+        {
+            if (!parser.writeLineF("{}\n", n ? 'T' : 'F'))
+                return false;
+        }
+        return true;
+    });
+    registerSerialiser<Array2D<double>>([](const std::any &a, LineParser &parser, const CoreData &coreData) {
+        const auto &v = std::any_cast<const Array2D<double> &>(a);
+        if (!parser.writeLineF("{}  {}  {}\n", v.nRows(), v.nColumns(), DissolveSys::btoa(v.halved())))
+            return false;
+        for (auto &n : v)
+        {
+            if (!parser.writeLineF("{}\n", n))
+                return false;
+        }
+        return true;
+    });
+    registerSerialiser<Array2D<std::vector<double>>>([](const std::any &a, LineParser &parser, const CoreData &coreData) {
+        const auto &v = std::any_cast<const Array2D<std::vector<double>> &>(a);
+        if (!parser.writeLineF("{}  {}  {}\n", v.nRows(), v.nColumns(), DissolveSys::btoa(v.halved())))
+            return false;
+        for (auto &data : v)
+        {
+            if (!parser.writeLineF("{}\n", data.size()))
+                return false;
+            for (auto &n : data)
+                if (!parser.writeLineF("{:16.9e}\n", n))
+                    return false;
+        }
+        return true;
+    });
+    registerSerialiser<Array2D<Data1D>>([](const std::any &a, LineParser &parser, const CoreData &coreData) {
+        const auto &v = std::any_cast<const Array2D<Data1D> &>(a);
+        if (!parser.writeLineF("{}  {}  {}\n", v.nRows(), v.nColumns(), DissolveSys::btoa(v.halved())))
+            return false;
+        for (auto &n : v)
+            if (!n.serialise(parser))
+                return false;
+        return true;
+    });
+    registerSerialiser<AtomTypeList>([](const std::any &a, LineParser &parser, const CoreData &coreData) {
+        return std::any_cast<const AtomTypeList &>(a).serialise(parser);
+    });
```

Comment:
```suggestion
    registerSerialiser<AtomTypeList>(simpleSerialise<AtomLtypeList>);
```

Like my suggestion with the deserialiser, this would require making another function

```
template <class T>
bool simpleSerialise(const std::any &a, lineParser &parser, const CoreData &coreData) {
    return std::any_cast<const T &>(a).serialise(parser);
}
```